src/_scss/pages/account/filters/programActivity/programActivity.scss

```diff
@@ -0,0 +1,31 @@
+.account-program-activity-filter {
+    padding-left: 0;
+    margin: 0.75rem 2.0rem 1rem;
+
+    > div {
+        padding-left: 0rem;
+    }
+
+    .shown {
+        min-height: 18rem;
+    }
+
+    ul.program-activities {
+        @include unstyled-list;
+
+        .program-activity-set {
+            @import "../../../search/filters/awardType/_sharedAwardStyle";
+            @include sharedAwardStyle;
```

Comment:
Can you middle or top-align the text to the checkboxes instead of bottom aligning them?